tests/commons/glob/test_glob.py

```
@@ -100,7 +100,7 @@ def visitDirectory(self, path: Path, applying: list[Any]) -> Any:
     }
 
     results = visitGlobPaths(
-        visitor, root, includes=["**/*"], applying=applying
+        visitor, root, includes="**/*", applying=applying
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)